CLAUDE-mobile.md

```
@@ -0,0 +1,201 @@
+# VOICEISOLATE PRO — Mobile Quick Reference
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)